js/src/components/EllipseGraphNode.vue

```diff
@@ -155,6 +197,7 @@ export default class EllipseGraphNode extends Vue {
   fitText() {
     Vue.nextTick(() => {
       this.computeWidthAndHeight();
+      console.log("max width: " + this.maxWidth);
```

Comment:
We probably don't want console statements in the code.

Reply:
done